src/deepsparse/open_pif_paf/utils/annotate.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    import openpifpaf
+
+    openpifpaf_error = None
+except ModuleNotFoundError as openpifpaf_import_error:
+    openpifpaf = None
+    openpifpaf_error = openpifpaf_import_error
```

Comment:
where is this used or raised?

Reply:
Yeah, this is not correct here.
What is the convention around making sure that we are setting up properly all the dependencies for the pipeline?
Proper edits in `setup.py` and pipelines `__init__.py`?

Reply:
My understanding is:
1. have feature in setup.py to install dependencies as you mention
2. in openpifpaf/__init__.py do a check to see if it's installed, and raise error if not
3. (optional) auto install dependencies in openpifpaf/__init__.py if not installed